admin/commands/status.go

```diff
@@ -41,6 +41,7 @@ PMM Client:
 	Connected        : {{ .PMMAgentStatus.Connected }}{{ if .PMMAgentStatus.Connected }}
 	Time drift       : {{ .PMMAgentStatus.ServerClockDrift }}
 	Latency          : {{ .PMMAgentStatus.ServerLatency }}{{ end }}
+	Up connected time: {{ .PMMAgentStatus.UpConnectedTime }}
```

Comment:
How about renaming it to `Uptime`?

Reply:
@ritbl uptime imples how long the agent has been up, right? This is not that metric

Reply:
I think it makes sense to have `Uptime For last 24h` is a value from 0 to 100
For example: we started at 12:00:00, now it is 20:00:00, we have 2 failed connection attempts recorded 14:00:00 and 15:44:10

Uptime is: 99.930%
( (8h - 20sec) / 8h ) * 100